ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -1247,6 +1247,14 @@ periodics:
               test:
               - ref: clusterbot-wait
               workflow: openshift-e2e-$(CLUSTER_TYPE)-ovn-dualstack
+          - as: launch-dualstack-primaryv6
+            steps:
+              cluster_profile: vsphere
+              env:
+                CLUSTER_DURATION: $(CLUSTER_DURATION)
+              test:
+              - ref: clusterbot-wait
+              workflow: openshift-e2e-$(CLUSTER_TYPE)-ovn-dualstack-primaryv6
```

Comment:
I am confused, where do we have `ovn-dualstack-primaryv6` workflow? AFAIK there is only `ovn-dualstack` workflow and it accepts ENV variable that configures the primary stack

Reply:
```
              env:
                CLUSTER_DURATION: $(CLUSTER_DURATION)
                my-ip-stack: that's-primary-v6
              test:
              - ref: clusterbot-wait
              workflow: openshift-e2e-$(CLUSTER_TYPE)-ovn-dualstack
```

something like that is what you need